tools/docs/platforms.ts

```diff
@@ -16,8 +23,7 @@ export async function generatePlatforms(dist: string): Promise<void> {
 
   platformContent += '.\n';
 
-  const indexFileName = `docs/usage/modules/platform/index.md`;
```

Comment:
unrelated change, but ok.

Reply:
This made the platform.ts consistent with manager.ts and datasources.ts